mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -72,27 +116,44 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 		ARG 2 y
 		ARG 3 z
 	METHOD method_10070 rotate (Lnet/minecraft/class_2470;)Lnet/minecraft/class_2338;
+		ARG 1 rotation
 	METHOD method_10071 unpackLongY (J)I
+		ARG 0 y
 	METHOD method_10072 south ()Lnet/minecraft/class_2338;
 	METHOD method_10074 down ()Lnet/minecraft/class_2338;
+	METHOD method_10075 crossProductPos (Lnet/minecraft/class_2382;)Lnet/minecraft/class_2338;
```

Comment:
Methods are allowed to overload by return type.

Reply:
The argument is still `Vec3i`, which means they differ only through their return type. This is only possible in bytecode, not at source level, so they must have different names in the source.

Reply:
`BlockPos` extends `Vec3i`, so at source level this is a method with a covariant return type. There is a similar synthetic method in `BlockPos` which does override the superclass method. Enigma doesn't allow covariant returns for whatever reason, though, so if we want it to have the same name (which I believe it should), it will have to be done manually.

Reply:
Oh, there is a bridge, should have said so from the start. Yes, it should definitely be called `crossProduct` then.